project/settings.py

```diff
@@ -99,6 +99,10 @@ def get_required_env(key: str) -> str:
     r"https://deploy-preview-(?:\d{1,4})--wow-django-dev\.netlify\.app",
 ]
 
+CORS_ALLOWED_ORIGIN_REGEXES = [
```

Comment:
looks like this is a duplicate of one of the regexes above?

Reply:
ah thanks, got duplicated in a merge update